plugins/components/rotation/Rotation.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** Rotation.cpp file
+*/
+
+#include "Rotation.hpp"
+#include "library_entrypoint.hpp"
+
+LIBRARY_ENTRYPOINT
+Components::IComponent *buildDefault()
+{
+    return new Components::Rotation();
+}
+
+LIBRARY_ENTRYPOINT
+Components::IComponent *buildWithParams(int32_t x, int32_t y, int32_t z)
```

Comment:
Same here

Reply:
Same thing, a rotation can be negative and positive ^^